docs/activedocs.md

```diff
@@ -42,17 +42,6 @@ OPTIONS
                                        ActiveDocs
        --skip-swagger-validations      Specify it to skip validation of the
                                        Swagger specification
-
```

Comment:
Where this docs pending to be regenerated before this PR? I see several removals for different docs

Reply:
Those removals do not belong to this PR. They are common options, not for the specific command being documented, but for the base commands. I think the options for the base commands is not necessary to document here. I added a new option to the top level command `3scale` and I had to update all the docs for all the subcomands. So I decided to remove. 

To be clear, the base command options are still printed when you invoke help, it is just removed from the docs.